ROM.sh

```diff
@@ -42,6 +42,12 @@ function cherrypick() # Automated Use only
     echo -e "${CL_WYT}==================================================================${NONE}";
 } # cherrypick
 
+function interrupt()
+{
+    echo -en "\n*** Ouch! Plz don't kill me! ***\n"
```

Comment:
1. Semicolon at the end
2. Remove `-n` from `echo`'s parameters, so that `\n` at the start of statement is made use of